packages/main/test/pages/AINoticeIndicator.html

```diff
@@ -0,0 +1,210 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no">
+    <meta charset="utf-8">
+    <title>AINoticeIndicator</title>
+    <script src="%VITE_BUNDLE_PATH%" type="module"></script>
+
+    <script>
+        document.addEventListener("DOMContentLoaded", function () {
+            const allPopovers = document.querySelectorAll("ui5-popover");
+            const allAINoticeIndicators = document.querySelectorAll("ui5-ai-notice-indicator");
+
+            allAINoticeIndicators.forEach((aiNotice) => {
+                aiNotice.addEventListener('click', (e) => {
```

Comment:
This handler is called when close button of the popover is clicked. Also, please use "ui5-" prefix for webcomponents events.